app/components/map/clusters/clusterMarker.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+  View,
+  Text
+} from 'react-native';
+import MapView from 'react-native-maps';
+import styles from './styles';
+
+function ClusterMarker(props) {
```

Comment:
Nice idea! we can improve the performance of that marker, let's use this as reference https://github.com/airbnb/react-native-maps/issues/741#issuecomment-279867763 later